models/DataObject/Service.php

```diff
@@ -47,8 +47,14 @@
  */
 class Service extends Model\Element\Service
 {
-    protected array $_copyRecursiveIds;
+    /**
+     * @internal
+     */
```

Comment:
Hi @blankse the PR looks good, but why do you want to add this internal notations?

Reply:
@robertSt7 The _ prefix in the name is a old convention for internal attributes, right? Otherwise it is also set in the Asset Service. I think this should be consistent in all 3 Services.